app/main/service_update_audits.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+from flask import render_template, redirect, url_for, request, session
+from . import main
+from .. import data_api_client
+from .helpers.auth import is_authenticated
+from .forms import ServiceUpdateAuditEventsForm
+from dmutils.audit import AuditTypes
+
+
+@main.route('/service-updates', methods=['GET'])
+def service_update_audits():
+    form = ServiceUpdateAuditEventsForm(request.args, csrf_enabled=False)
+
+    if form.validate():
+        audit_events = data_api_client.find_audit_events(
+            audit_type=AuditTypes.update_service.value,
+            acknowledged=form.default_acknowledged(),
+            audit_date=form.format_date()
+        )
+
+        return render_template(
+            "service_update_audits.html",
+            today=form.format_date_for_display(),
+            acknowledged=form.default_acknowledged(),
+            audit_events=audit_events['auditEvents'],
+            form=form,
+            **get_template_data())
+    else:
+        return render_template(
+            "service_update_audits.html",
+            today=datetime.now().strftime("%d/%m/%Y"),
```

Comment:
I think the formatting of the date should be done in the template with the newly created `dateformat` template filter.